en/08_Changelogs/6.1.0.md

```diff
@@ -99,6 +100,27 @@ If you are using the [`DataObjectSchema::databaseIndexes()`](api:SilverStripe\OR
 
 See the [indexes documentation](/developer_guides/model/indexes/) for more details.
 
+### Generated column support {#generated-columns}
+
+Generated columns are database columns where the value is generated inside the database, rather than being set by a user. They're usually based on other columns in the database and can be either generated when the record is updated and stored, or generated when requested in which case they aren't stored.
+
+Some good use cases for generated columns include:
+
+- sorting in a gridfield on a complex summary field: It's common to use a getter method to get some value derived from your database fields (e.g. a discounted price), and use that in `summary_fields`. With a generated column, you can remove the getter method and the [`GridFieldSortableHeader`](api:SilverStripe\Forms\GridField\GridFieldSortableHeader) component will be able to sort using the column.
+- using functional indexes: generated columns can be included in indexes, allowing you to sort or filter by complex expressions in an efficient way.
+- data integrity: unlike generating values inside an `onBeforeWrite()` extension hook, generated column values will be correct even if you update the record with raw SQL.
```

Comment:
```suggestion
- data integrity: unlike generating values inside an `onBeforeWrite()` extension hook, generated column values will be correct even if you update the record with raw SQL.
```

Probably also restoring old versions of records and archived records which have had columns added since the old record was created

Reply:
Done